plugins/ps/pre-deploy

```diff
@@ -5,4 +5,5 @@ source "$PLUGIN_AVAILABLE_PATH/ps/functions"
 
 APP="$1"
 
+get_procfile $APP
```

Comment:
We don't need a herokuish app's Procfile. Can we only extract the `Procfile` in the necessary case?